tests/cypress/support/stateSubmissionFormCommands.ts

```diff
@@ -209,6 +209,9 @@ Cypress.Commands.add('submitStateSubmissionForm', (success = true) => {
     cy.findAllByTestId('modalWindow')
         .should('exist')
         .within(($modal) => {
+            if (resubmission) {
+                cy.get('#submittedReasonCharacterCount').type('Resubmission summary')
+            }
             cy.findByTestId('review-and-submit-modal-submit').click()
```

Comment:
I added this part for resubmissions. Let me know if it looks ok.

Reply:
yes I saw that! looks good